modules/javafx.graphics/src/main/java/com/sun/javafx/application/PlatformImpl.java

```diff
@@ -715,6 +716,34 @@ public static void setPlatformUserAgentStylesheet(final String stylesheetUrl) {
         }
     }
 
+    public enum HighContrastScheme {
+        HIGH_CONTRAST_BLACK("high.contrast.black.theme"),
+        HIGH_CONTRAST_WHITE("high.contrast.white.theme"),
+        HIGH_CONTRAST_1("high.contrast.1.theme"),
+        HIGH_CONTRAST_2("high.contrast.2.theme");
+
+        private final String themeKey;
+        HighContrastScheme(String themeKey) {
+            this.themeKey = themeKey;
+        }
+
+        public String getThemeKey() {
+            return themeKey;
+        }
+
+        public static String fromThemeName(Function<String, String> keyFunction, String themeName) {
```

Comment:
Would be good to get a bit of (java)doc here so that it is clear what this function is doing.

Reply:
Ok, makes sense. Done